lib/configs/angular.js

```diff
@@ -99,10 +98,16 @@ module.exports = {
         }],
         'space-before-blocks': [2, "always"],
         'consistent-this': [2, "self"],
-        'indent': [2, 4, {
-            SwitchCase: 1
-        }],
-        quotes: [2, "single"]
+        quotes: [2, "single"],
+        'prefer-const': 1,
+        'prefer-template': 1,
```

Comment:
"will replace concatenation with template strings" <- this was the piece I was curious about.

With that condition, and knowing that we will be babel-ing our code I'm good with it. :+1: